### PR TITLE
application: system librarian role

### PR DESCRIPTION
**Pull request by Garfield-fr**
* NEW Adds system librarian role
* NEW Updates roles enum on patron record and patron form

Signed-off-by: Bertrand Zuchuat <bertrand.zuchuat@rero.ch>

How to test:
- `./script/bootstrap`
- `./script/setup`
- Check user roles on list and form

### CODE REVIEW

**Comment by BadrAly** [comment]
must rebase with `dev` after merging https://github.com/rero/rero-ils/pull/260

File: rero_ils/modules/patrons/jsonschemas/form_patrons/patron-v0.0.1.json [hunk 1]
```diff
@@ -74,13 +74,23 @@
           {
             "key": "roles",
             "notitle": "true",
-            "description": "Is the user a patron or/and a librarian?",
+            "description": "User role(s) ?",
```
Comment (iGormilhit) [inline rero_ils/modules/patrons/jsonschemas/form_patrons/patron-v0.0.1.json]: `"description": "What is/are the role(s) of the user?"`

File: rero_ils/modules/patrons/jsonschemas/form_patrons/patron-v0.0.1.json [hunk 2]
```diff
@@ -74,13 +74,23 @@
...
+                "value": "patron",
+                "name": "patron"
+              },
+              {
+                "value": "librarian",
+                "name": "librarian"
+              },
+              {
+                "value": "systemlibrarian",
+                "name": "systemlibrarian"
```
Comment (iGormilhit) [inline rero_ils/modules/patrons/jsonschemas/form_patrons/patron-v0.0.1.json]: `"name": "system librarian"`

Else, why a a titlemap if the values are all equal names?

Comment (Garfield-fr) [reply rero_ils/modules/patrons/jsonschemas/form_patrons/patron-v0.0.1.json]: To allow translation

File: rero_ils/modules/patrons/jsonschemas/form_patrons/patron-v0.0.1.json (hunk 1 above)
Comment (Garfield-fr) [reply rero_ils/modules/patrons/jsonschemas/form_patrons/patron-v0.0.1.json]: done

**Pull request closed without merging**